application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -155,38 +157,40 @@ private boolean isAuthorOnCooldown(long userId) {
             .isPresent();
     }
 
-    private MessageCreateAction createModMessage(String reportReason, User author,
-            String reportedMessage, String reportedMessageUrl, String reportedMessageTimestamp,
-            TextChannel modMailAuditLog) {
+    private MessageCreateAction createModMessage(String reportReason, String authorName,
+            String authorAvatarURL, String authorID, String reportedMessage,
+            Instant reportedMessageTimestamp, TextChannel modMailAuditLog) {
         MessageEmbed reportedMessageEmbed = new EmbedBuilder().setTitle("Report")
-            .setDescription("""
-                    Reported Message: **%s**
-                    [Reported Message URL](%s)""".formatted(reportedMessage, reportedMessageUrl))
-            .setAuthor(author.getName(), null, author.getAvatarUrl())
-            .setTimestamp(Instant.parse(reportedMessageTimestamp))
+            .setDescription("Reported Message: **%s**".formatted(
+                    MessageUtils.abbreviate(reportedMessage, MessageEmbed.DESCRIPTION_MAX_LENGTH)))
+            .setAuthor(authorName, null, authorAvatarURL)
+            .setTimestamp(reportedMessageTimestamp)
             .setColor(AMBIENT_COLOR)
             .build();
 
         MessageEmbed reportReasonEmbed = new EmbedBuilder().setTitle("Reason")
             .setDescription(reportReason)
             .setColor(AMBIENT_COLOR)
             .build();
-        return modMailAuditLog.sendMessageEmbeds(reportedMessageEmbed, reportReasonEmbed);
+        return modMailAuditLog.sendMessageEmbeds(reportedMessageEmbed, reportReasonEmbed)
+            .addActionRow(
+                    DiscordClientAction.General.USER.asLinkButton("Author Profile", authorID));
```

Comment:
u added the wrong button. the idea was to have a button that jumps to the message. not to the author profile^^